tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -663,6 +663,34 @@ def image_exists(self, imagespec):
 
         return 0
 
+    def rename_primary_image(self, source_imagespec, dest_imagespec, peercluster, **kw):
+        """
+        Rename the primary image and check from secondary for the changes
+        Args:
+           source_imagespec: primary image
+           dest_imagespec: rename image
+           peercluster: secondary cluster
```

Comment:
Why do we need peer cluster in this method?
We are not using it right ?

Reply:
Forgot to remove. I will remove it.